### PR TITLE
add account interest concept

### PR DESCRIPTION
**Pull request by kluver**

### CODE REVIEW

File: src/poprox_concepts/domain/account.py [hunk 1]
```diff
@@ -6,3 +6,10 @@
 class Account(BaseModel):
     account_id: UUID = None
     email: str
+
+
+class Account_Interest(BaseException):
+    account_id: UUID = None
+    entity_id: UUID
+    preference: int
+    frequency: int
```
Comment (karlhigley) [inline src/poprox_concepts/domain/account.py:15]: Seems like this should maybe also have the name of the entity for ease of use

**Review by karlhigley — APPROVED** [review]
Doesn't need to be in this PR, but now that we have multiple concepts related to people's interests (`AccountInterest` and `ClickHistory`), it might be time for us to create a `UserProfile` class.

File: src/poprox_concepts/domain/account.py [hunk 2]
```diff
@@ -6,3 +6,10 @@
 class Account(BaseModel):
     account_id: UUID = None
     email: str
+
+
+class Account_Interest(BaseException):
```
Comment (karlhigley) [inline src/poprox_concepts/domain/account.py:11]:
```suggestion
class AccountInterest(BaseModel):
```

Comment (kluver) [reply src/poprox_concepts/domain/account.py:11]: Doh!

**Pull request merged**